Adds military helipad location

CountAlex opened the pull request:
#### Summary
SUMMARY: Content "Adds new mapgen location - military helipad"

#### Purpose of change
New location.

#### Describe the solution
This is small military helicopter pad outpost. Nothing like full-scale helicopter base (those are rare) or something like that.
Just a small military location with 3 pads, hangar, barracks and some guarding turrets.

#### Testing
Main level - pads, offices, cantina, warehouse, parking lot and entrance.
![main](https://user-images.githubusercontent.com/26584783/71895113-317bb100-3161-11ea-8ab7-8f01939d5df8.png)

2nd level - barracks and roofs.
![roof](https://user-images.githubusercontent.com/26584783/71895124-3b9daf80-3161-11ea-8bed-bfbc4fc622ab.png)

#### Additional context
I will add screenshots so people might have a better look.
This also adds outdoor version of yellow concrete so pads can be accented.

curstwist commented:
screenshots please.

Amoebka commented:
This should probably have a single-frame crane vehicle instead of the guaranteed lifter mech. Those are rare prototypes, not standart issue equipment for small military installations.

CountAlex commented:
> screenshots please.
Let me weed out errors, and I will throw them in.

> This should probably have a single-frame crane vehicle instead of the guaranteed lifter mech. Those are rare prototypes, not standart issue equipment for small military installations.
Then I will figure something out. Forklift car might be a good solution.

I-am-Erk commented on data/json/mapgen_palettes/helipad.json:
```diff
@@ -0,0 +1,76 @@
+      "p": "t_pavement",
+      "m": "t_door_metal_c",
+      "y": "t_pavement_y",
+      ".": "t_grass",
```
```suggestion
      ".": "t_region_groundcover_urban",
```

I-am-Erk commented on data/json/mapgen_palettes/helipad.json:
```diff
@@ -0,0 +1,76 @@
+      "m": "t_door_metal_c",
+      "y": "t_pavement_y",
+      ".": "t_grass",
+      "T": "t_grass",
```
```suggestion
      "T": "t_region_groundcover_urban",
```

I-am-Erk commented on data/json/mapgen/helipad.json:
```diff
@@ -0,0 +1,215 @@
+      "items": {
+        "B": {
+          "item": {
+            "subtype": "distribution",
```
This seems like something that could be defined in an itemgroup, perhaps in supplies.json. I can see other people having use for drums of gasoline and motor oil

CountAlex replied:
I've added it as you suggested. I will give it some test about spawned amounts - I am not sure how charges work with fuels exactly. Besides I'd like to have some opinion on how much fuel spawned barrels should have. 
I also think motor oil might need to have different container than plastic jerrycan.

I-am-Erk commented on data/json/mapgen/helipad.json:
```diff
@@ -0,0 +1,215 @@
+        "O": { "item": "SUS_oven" },
+        "H": { "item": "SUS_spice_collection" }
+      },
+      "place_monsters": [
```
I don't know the groups well but I'd assume the regular, non-weak military groups should also have a chance of spawning here if it's a major outpost with a helipad.

CountAlex replied:
Tweaked that a bit. 
There are three military monstergroups, GROUP_MIL_WEAK, GROUP_MIL_STRONG and GROUP_MAYBE_MIL. First two have different sizes and stronger Zs in latter case, third one also adds military robots to the mix.

CountAlex commented:
Cleaned up some things and added screenshots. 
Also moved copters spawning points a bit and decreased chance of them spawning since they spawn way too intact without making it hardcoded as it works for map extras.

Amoebka commented:
10 turrets is 16.000 rounds of 5.56 a player can loot from one location.
Just saying.

I-am-Erk commented:
The ammo loot isn't really a consideration when designing a map. That's a problem with calculating turret age, not mapgen.

Amoebka commented:
Yes, but mapgen can still take it into account if there are no downsides.

10 turrets don't exactly guard the location any better than 4. Gameplay-wise turrets are gear checks, and a player that can deal with 4 turrets can just as easily deal with 10, the only difference being the ammo reward. Story-wise, this is a small refueling base deep inside home territory, it doesn't need multiple machinegun nests with overlapping firing arcs. A few turrets would do just fine to deter theoretical terrorists and random idiots.

CountAlex commented:
Changing amount of turrets will take me two minutes while this is still at PR stage. Just tell me if I need to.

Amoebka commented:
I would say leave just the 4 turrets in the corners. Their ranges should be high enough to cover the entire perimeter anyway.

CountAlex commented:
Yeah, I guess it makes sense. Missile silo also has only corner ones (and couple near entrance) while being way more significant military object.
